docs/languages/java.md

```diff
@@ -72,8 +72,20 @@ This document describes some of the key features included in those Java extensio
 
 **NOTE:** If you are using VS Code on Windows and want to take advantage of the Windows Subsystem for Linux, see [Developing in WSL](/docs/remote/wsl).
 
-Before you start, you must have the Java Development Kit (JDK) on your local environment to run your projects.
-For how to configure these versions, refer to [Configure Runtime for Projects](/docs/java/java-project.md#configure-runtime-for-projects).
+### Install a Java development kit (JDK)
+
+Java development kit (JDK) is a software development environment used for developing Java applications. In order to run Java within Visual Studio Code, you need to install a JDK. The [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack) supports Java version 1.5 or above.
```

Comment:
trying to make this as obvious as possible. For professionals it doesn't matter because they already know what JDK is, this is more for students